test/OpenTelemetry.Tests/OpenTelemetry.Tests.csproj

```diff
@@ -18,6 +18,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" />
     <PackageReference Include="Microsoft.CSharp" Condition="'$(TargetFramework)' == 'net462'" />
+    <PackageReference Include="System.Text.Json" />
```

Comment:
(nit) Is this actually needed? Since the tests reference the product, which references System.Text.Json, this reference should come in transitively.

Reply:
I think it's necessary. `OpenTelemetry` itself doesn't have `System.Text.Json` dependency. The new helper is actually only used by some of the exporter packages, which do have S.T.J dependency, but the source code lives in the "Shared" folder now, and the tests for the "Shared" code are currently in this project.

Reply:
@Yun-Ting If we add `Condition="'$(TargetFramework)' == 'net462'"` does it compile? The working theory was if you reference `net6.0`+ then STJ would automatically be available so that should confirm 😄